shared/test/pageStore_test.js

```diff
@@ -90,6 +90,31 @@ describe("loop.store.PageStore", () => {
     });
   });
 
+  describe("UpdatePage", () => {
+    beforeEach(() => {
+      let pages = [{ id: "fakeId" }];
+      sandbox.stub(store, "getStoreState").withArgs("pages").returns(pages);
+      sandbox.stub(store, "setStoreState");
+    });
+
+    it("should correctly update page matching the id", () => {
+      let updatedPage = {
+        pageId: "fakeId",
+        thumbnail_img: "fakeThumb",
+        title: "fakeTitle"
+      };
+
+      store.updatePage(updatedPage);
```

Comment:
Create an action instead of a simple object. As we do in [https://github.com/mozilla/loop/pull/546/files#diff-fce74e07540ed8189a559e9b22cf3243L86](line 86)